### DISCUSSION TITLE
Highlight another area on hover/click

### DISCUSSION THREAD

**Discussion by L2Develop96** [img-mapper/img-mapper#55] [Q&A] [answered]
Hi,

I want to hightlight two different areas when I hover on one of them. And once I click on one of them the highlight stays enabled.
I suggest the multiple select of different area should be supported in this module because it's really useful!

Any ideas on how to get his done? thanks.

**Comment by NishargShah** [comment, marked as answer]
That's a good idea but as of now, you can achieve it by adding `preFillColor` property when you click/hover on one of the items.

Note: You need to turn off the `active` feature because your current state and mapper state crossing each other and you will notice some problems due to that.

Codesandbox: https://codesandbox.io/s/billowing-bush-u4s9jb

**Reply by L2Develop96** [reply]
Thank you for reply. I've tried your sandbox and yes it is working correctly but  it's not holding the highlighted areas on click. I mean it should be saved in the state right?

Is there any way to hold the highlighted elements on click (Both areas)?

**Reply by NishargShah** [reply]
I changed some of the things to hold the highlighting when you click

Please check: https://codesandbox.io/s/billowing-bush-u4s9jb

**Reply by L2Develop96** [reply]
Works like charm, thank you!
